solr/core/src/java/org/apache/solr/handler/SnapShooter.java

```diff
@@ -272,9 +273,11 @@ public void createSnapAsync(final int numberToKeep, Consumer<NamedList<?>> resul
         solrCore.getDirectoryFactory().release(dir);
       }
 
-      details.add("fileCount", files.size());
+      details.add("fileCount", files.size()); // DEPRECATED: for removal, replaced with indexFileCount
+      details.add("indexFileCount", files.size());
       details.add("status", "success");
-      details.add("snapshotCompletedAt", new Date().toString());//bad; should be Instant.now().toString()
+      details.add("snapshotCompletedAt", Instant.now().toString()); // DEPRECATED: for removal, replaced with endTime
+      details.add("endTime", Instant.now().toString());
```

Comment:
Maybe have a local to ensure both timestamps are 100% identical and/or indicate intended removal timeline in the comment e.g.
```suggestion
      final String endTime = Instant.now();
      details.add("snapshotCompletedAt", endTime); // DEPRECATED: for removal in Solr 10, replaced with endTime
      details.add("endTime", endTime);
```

Reply:
Yeah, sure let's extract to local variable. Concerning sunset time, not sure what period is usually applied. One major release?

Reply:
> Maybe have a local to ensure both timestamps are 100% identical

Done